tasks/extract_orgs.yml

```diff
@@ -44,11 +49,9 @@
       mode: 0400
       src: key.pem.j2
   - name: Generate a UUID for this Receptor node
-    command: "uuidgen"
-    register: uuidgen_output
-  - name: Set Receptor Node ID fact
-    set_fact:
-      receptor_uuid: "{{ uuidgen_output.stdout }}"
+    shell: "uuidgen > {{ receptor_config_dir }}/rh_{{ sat_account_id }}/uuid"
```

Comment:
Is uuidgen always installed?

Reply:
It's part of util-linux. It's safe to say it's installed on a RHEL box.